src/mongo/client/dbclientinterface.h

```diff
@@ -1248,6 +1254,13 @@ namespace mongo {
         virtual void _auth(const BSONObj& params);
 
         /**
+         * Create and return an MD5 digest for authentication for copydb.
+         */
+        std::string _authKeyCopyDB(const std::string &username,
+                                   const std::string &password,
```

Comment:
if this function is not used elsewhere (or unit tested), can we just put it in an anonymous namespace and get rid of this declaration?